source/server/spongineer/properties.rst

```diff
@@ -0,0 +1,74 @@
+| **timings.bypassMax**              | Allows for bypassing the max number of frames to send to the Aikar's timing     |   
+|                                    | API. Setting this will not permit bypassing the limit unless the API is         |  
+| (any value that exists)            | configured to permit it.                                                        |
++------------------------------------+---------------------------------------------------------------------------------+
+|                                    | Adds a StackTrace to each ``StackFrame`` for when either the Sponge             |
+| **sponge.debugcauseframes**        | implementation or a plugin potentially corrupts the ``CauseStackManager`` 's    |
+|                                    | ``StackFrame`` s, whether it's a frame being popped out of order, or popping    |
+| (boolean, default: false)          | an already popped frame. This is useful to find out where the frame was         | 
+|                                    | created.                                                                        |
++------------------------------------+---------------------------------------------------------------------------------+
+|                                    | Similar to the ``StackFrame`` exception creation, this tracks the stack trace   |
+|                                    | for each ``BlockTransaction`` that is captured for a block being changed, a     | 
+| **sponge.debugProxyChanges**       | neighbor notification request, or a ``TileEntity`` change. Useful to find out   |
+|                                    | where each transaction is created, potentially adding more information if there |
+| (boolean, default: false)          | is an exception thrown during the processing of said transactions. Should       |
+|                                    | always be included for logs when reporting issues involving these sort of       |
+|                                    | transactions.                                                                   |
++------------------------------------+---------------------------------------------------------------------------------+
+| **sponge.filter.debug**            | Writes out classes that are generated by the filter generator to the            |
+|                                    | ``.sponge.debug.out`` folder, named after the class they are altering.          |
+| (boolean, default: false)          |                                                                                 |
++------------------------------------+---------------------------------------------------------------------------------+
+| **sponge.print_all_catalog_types** | Prints out all registered ``CatalogType`` s and their ``String`` id's at the    | 
+|                                    | end of startup, useful for finding out what id's are registered from vanilla,   |
```

Comment:
Should be a semicolon (or full stop and new sentence) after `startup`. Or add `which is`

Reply:
Added `which is`